src/Migrate/custom/New-AzMigrateProject.ps1

```diff
@@ -15,59 +15,60 @@
 
 <#
 .Synopsis
-Get All servers in a migrate project.
+Get All servers in a migrate project. New proj.
 .Description
 Get Azure migrate server commandlet fetches all servers in a migrate project.
 .Link
 https://docs.microsoft.com/en-us/powershell/module/az.migrate/get-azmigrateserver
 #>
 
-function Get-AzMigrateServer {
-    [OutputType([Microsoft.Azure.PowerShell.Cmdlets.Migrate.Models.Api202001.IVMwareMachine])]
+function New-AzMigrateProject {
+    [CmdletBinding(DefaultParameterSetName='PutExpandedCustom', PositionalBinding=$false, SupportsShouldProcess, ConfirmImpact='Medium')]
     param (
-        [Parameter(Mandatory)]
+        [Parameter(ParameterSetName='PutExpandedCustom', Mandatory)]
         [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
         [System.String]
         # Specifies the migrate project name.
-        ${MigrateProjectName},
+        ${Name},
 
-        [Parameter(Mandatory)]
+        [Parameter(ParameterSetName='PutExpandedCustom', Mandatory)]
         [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
         [System.String]
         # Specifies the resource group name.
         ${ResourceGroupName},
 
+        [Parameter(ParameterSetName='PutExpandedCustom', Mandatory)]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
+        [System.String]
+        # Specifies the VMware machine name.
+        ${Location},
+
+        [Parameter(ParameterSetName='PutExpandedCustom')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
+        [System.String]
+        # Specifies the VMware machine name.
+        ${ETag},
+
+        [Parameter(ParameterSetName='PutExpandedCustom')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
+        [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Models.Api20180901Preview.IMigrateProjectProperties]
+        # Specifies the project properties.
+        ${Property},
+
         [Parameter()]
         [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Category('Path')]
         [Microsoft.Azure.PowerShell.Cmdlets.Migrate.Runtime.DefaultInfo(Script='(Get-AzContext).Subscription.Id')]
-        [System.String[]]
+        [System.String]
         # Specifies the subscription id.
         ${SubscriptionId}
```

Comment:
In PowerShell you can pass 1 single value to a parameter of a collection type. For example, for a `-SubscriptionId <string[]>` it's totally fine to `-SubscriptionId "one-string-instead-of-an-array"`. That's why we always model ${SubscriptionId} as an array, so please change it back

Reply:
Hi, In Az Migrate, we would like to operate on only one subscription ID at a time, and not on multiple, as our service has limitations. Hence this behavior.

Reply:
If you supply multiple sub ids, the cmdlet will send multiple requests, each of them to 1 sub, so service side will always receive only 1 sub id per request. 
(This is handled by generated code)

Reply:
But our service/product is not designed like that. So Migrate Project will use 1 Resource Group , 1 Subscription ID in it's life time. Our custom code will not know how to handle / collate information from different subscription ids, as there will be nothing common between them.

Reply:
Multiple sub ids, will be fundamentally multiple customers for us.

Reply:
OK got it!